Feature/publish integration for ms intune (#493)
appcircleio/appcircle-docusaurus: docs/publish-integrations/common-publish-integrations/send-to-microsoft-intune.md

---
title: Publish to Microsoft Intune

> description: Submit your Android application to the Microsoft Intune with Appcircle

Reviewer: ```suggestion
description: Learn how to submit your Android app to Microsoft Intune using Appcircle’s streamlined integration process.
```

- DeviceManagementApps.Read.All
- DeviceManagementApps.ReadWrite.All

You can find detailed information about granting permissions [here](https://learn.microsoft.com/en-us/entra/identity-platform/howto-call-a-web-api-with-curl?tabs=dotnet6&pivots=no-api#add-application-permissions-to-allow-access-to-a-web-api).
:::

### Input Variables

Below are the parameters necessary for this step's operation, along with their descriptions.

<Screenshot url='https://cdn.appcircle.io/docs/assets/send-to-microsoft-intune-inputs.png' />

| Variable Name        | Description                                                                                                                                                                                                                       | Status   |
| -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |

> | `$AC_INTUNE_AUTH_URL` | Specifies the URL generated by the application you created on the Microsoft Identity Platform for authorization. This URL should be in the following format: `https://login.microsoftonline.com/{tenant_id}/oauth2/v2.0/token`. You can find more details [here](https://learn.microsoft.com/en-us/entra/identity-platform/howto-call-a-web-api-with-curl?tabs=dotnet6&pivots=no-api#register-the-web-api).                                                                  | Required |

Reviewer: ```suggestion
| `$AC_INTUNE_AUTH_URL` | Specifies the authorization URL generated by the application you created on the Microsoft Identity Platform. This URL should be in the following format: `https://login.microsoftonline.com/{tenant_id}/oauth2/v2.0/token`. More details [here](https://learn.microsoft.com/en-us/entra/identity-platform/howto-call-a-web-api-with-curl?tabs=dotnet6&pivots=no-api#register-the-web-api).                                                            | Required |
```

| `$AC_INTUNE_CLIENT_ID`  | Specifies the [Application (client) ID](https://learn.microsoft.com/en-us/entra/identity-platform/howto-call-a-web-api-with-curl?tabs=dotnet6&pivots=no-api#register-the-web-api) which uniquely identifies your application in the Microsoft cloud ecosystem, across all tenants.                                                                                                                                                                                                      | Required |

> | `$AC_INTUNE_CLIENT_SECRET`     | A client secret is a string value your app can use to identity itself, and is sometimes referred to as an application password. You can learn how to create it [here](https://learn.microsoft.com/en-us/graph/auth-register-app-v2#option-2-add-a-client-secret).                                                                                                                                                                | Required |

Reviewer: ```suggestion
| `$AC_INTUNE_CLIENT_SECRET`     | A client secret, sometimes referred to as an application password, is a string value your app can use to identify itself. Learn how to create one [here](https://learn.microsoft.com/en-us/graph/auth-register-app-v2#option-2-add-a-client-secret).                                                                                                                                                   | Required |
```
